learning/tour-of-beam/frontend/lib/constants/colors.dart

```diff
@@ -18,16 +18,23 @@
 
 import 'package:flutter/material.dart';
 
-// Light Theme base colors
-const Color kLightPrimaryBackground = Colors.white;
-const Color kLightSecondaryBackground = Color(0xFFFCFCFC);
-const Color kLightGrey = Color(0xFFE5E5E5);
-const Color kLightText = Color(0xFF242639);
-const Color kLightPrimary = Color(0xFFE74D1A);
+class ProjectColors {
```

Comment:
Feels redundant. Why defining these two constants?

Reply:
I was asked to store all colors in one place in previous projects in order to remove or replace them more easily. Should I remove these black and white constants?

Reply:
Feels OK now, thanks.